<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>791</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>791</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>791</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bulk Rename System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="appNameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Bulk File Rename System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="creditsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1021</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Made By: Adarsh Kumar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#1e90ff;&quot;&gt;ðŸª¶ðŸ‡§ðŸ‡«ðŸ‡«ðŸ‡·ðŸ‡¸&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEntry">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="extensionEntry">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="extensionButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Change Extension</string>
    </property>
   </widget>
   <widget class="Line" name="hrLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1001</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListView" name="listview">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>451</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListView" name="selectView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>441</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Side Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Side Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="chooseLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>1021</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Choose Any One Operation &amp;amp; Apply Changes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="addfirstRadio">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add First</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="removefirstRadio">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>650</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove First</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="addlastRadio">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Last</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="removelastRadio">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>650</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Last</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="newnameRadio">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>650</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add NewName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>690</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 144, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apply Changes/Rename</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="renameEntry">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>690</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="renameMsgLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>690</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rename Here:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
